packs/october.js

```diff
@@ -0,0 +1,39 @@
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by removing unused CSS, in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'unused-css-rules': "Consider reviewing the [plugins](https://octobercms.com/plugins) loading unused CSS on your site. To identify plugins that are adding unnecessary CSS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the stylesheet. Look out for plugins that have many stylesheets in the list which have a lot of red in code coverage. A plugin should only enqueue a stylesheet if it is actually used on the page.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using webp in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-webp-images': "Consider using a [plugin](https://octobercms.com/plugins?search=image) or service that will automatically convert your uploaded images to the optimal formats.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by lazy loading images that are initially offscreen in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'offscreen-images': "Consider installing a [image lazy-loading plugin](https://octobercms.com/plugins/) that provides the ability to defer any offscreen images, or switch to a theme that provides that functionality. Also consider using [the AMP plugin](https://octobercms.com/plugins?search=Accelerated+Mobile+Pages).",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site loading performance by reducing the total bytes delivered by their page in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'total-byte-weight': "Consider showing excerpts in your post lists (e.g. via the more tag), reducing the number of posts shown on a given page, breaking your long posts into multiple pages, or using a plugin to lazy-load comments.",
```

Comment:
is "more tag" referring to a component of the CMS editor UI? Is this component translated (does October CMS support languages other than English)? If it is translated, let's mention that in the translator comment. They'll restranslate themselves but chances are it'll match w/e October  uses.

Reply:
`(e.g. via the more tag)` going to change into `(e.g. via a "show more" tag)`